logstash-core/src/main/java/org/logstash/ackedqueue/FileSettings.java

```diff
@@ -107,6 +114,9 @@ public String getDirPath() {
     }
 
     @Override
+    public long getQueueMaxBytes() { return queueMaxBytes; }
```

Comment:
I thought I was following the current style, but the IDE tricked me :D

<img width="452" alt="screen shot 2016-11-28 at 10 11 39" src="https://cloud.githubusercontent.com/assets/31809/20664428/1bb94c84-b553-11e6-8aa2-55c5eb91e42c.png">